src/web/handlers.rs

```diff
@@ -318,77 +318,55 @@ pub async fn post_collection_batch(
         .await?
     };
 
-    let commit = breq.commit;
     let user_id = coll.user_id.clone();
     let collection = coll.collection.clone();
 
     let mut success = vec![];
     let mut failed = coll.bsos.invalid;
+    // Option #2, mark each bso.id included in the "commit" as "failing" so that they're
```

Comment:
kill this stale comment, it's about the old version of this patch. the commit message also needs an update